### ISSUE TITLE
resume training problem

### ISSUE THREAD

**Issue by Roujack** [xw-hu/Mask-ShadowGAN#11]
Hi, I've try continue training from a checkpoint, and after several epoches the loss become very large(larger than 1w). I find that the learning rate of resume training is a value less than 0, which causes the above problem. I think the solution is to set the offset parameter of lambdaLR  to 0.
the relative code is:
train_Mask-ShadowGAN: line 97-101:
```
lr_scheduler_G = torch.optim.lr_scheduler.LambdaLR(optimizer_G,lr_lambda=LambdaLR(opt.n_epochs, 0, opt.decay_epoch).step)
lr_scheduler_D_A = torch.optim.lr_scheduler.LambdaLR(optimizer_D_A,lr_lambda=LambdaLR(opt.n_epochs, 0, opt.decay_epoch).step)
lr_scheduler_D_B = torch.optim.lr_scheduler.LambdaLR(optimizer_D_B,lr_lambda=LambdaLR(opt.n_epochs, 0,opt.decay_epoch).step)

```
